lib/logstash/inputs/twitter.rb

```diff
@@ -130,4 +193,30 @@ def from_tweet(tweet)
 
     event
   end
+
+  def configure(c)
+    c.consumer_key = @consumer_key
+    c.consumer_secret = @consumer_secret.value
+    c.access_token = @oauth_token
+    c.access_token_secret = @oauth_token_secret.value
+  end
+
+  def build_options
+    build_options = {}
+    build_options[:track]     = @keywords.join(",")  if @keywords && @keywords.length > 0
+    build_options[:locations] = @locations           if @locations && @locations.length > 0
+    build_options[:language]  = @languages.join(",") if @languages && @languages.length > 0
+
+    if @follows && @follows.length > 0
+      build_options[:follow]    = @follows.map do |username|
+        (  username.to_i == 0 ? find_userid(username) : username )
+      end.join(",")
+    end
+    build_options
+  end
+
+  def find_userid(username)
+    @rest_client.user(:user => username)
```

Comment:
I'm sorry, I don't understand your question here. Can you elaborate it a bit more?